src/test/java/com/citihub/configr/namespace/TestConfigurationService.java

```diff
@@ -201,244 +194,4 @@ public void testWouldNotReplace() throws Exception {
     assertThat(result).isFalse();
   }
 
-  @Test
```

Comment:
Moved into a new class, TestNSServiceValidations to keep the file from being so big